io.gitlab.jazztickets.aimtrain.json

```diff
@@ -0,0 +1,43 @@
+{
+	"app-id": "io.gitlab.jazztickets.aimtrain",
```

Comment:
this should be `io.gitlab.aimtrain.aimtrain`, rename https://gitlab.com/aimtrain/code to aimtrain or create another repo there with the same name

Reply:
the check runs only on new submissions